src/glacier_upload/delete_archive.py

```diff
@@ -17,3 +12,12 @@ def delete_archive(vault_name, archive_id):
         archiveId=archive_id)
 
     click.echo('Delete archive request sent.')
+
+#TODO: the argument name was wrong and wasnt working before
```

Comment:
I'm not sure I understand this TODO tag, what is there still not yet done?
By the way thanks for fixing the wrong argument name.